lib/configparser.py

```diff
@@ -96,16 +97,11 @@ def confparser(self, section, option):
 
     def checkSys(self, addr, name):
         '''Check for server availability'''
-        rc = os.system("ping -c 8 %s > /dev/null" % addr)
-        for x in range(6):
-            if rc:
-                rc = os.system("ping -c 1 %s > /dev/null" % addr)
-            else:
-                break
+        rc = os.system("ping -c 1 %s > /dev/null" % addr)
```

Comment:
Please remove this logic from commit , may before sending PR you miss to rebase e